lib/comictaggerlib/comicvinetalker.py

```diff
@@ -411,13 +417,15 @@ def fetchIssuesByVolumeIssueNumAndYear(
 
         issue_number = urllib.parse.quote_plus(str(issue_number).encode("utf-8"))
 
-        filter = "&filter=" + volume_filter + \
-            year_filter + ",issue_number:" + issue_number
+        filter_line = '%s%s,issue_number:' % (volume_filter, year_filter, issue_number)
```

Comment:
```suggestion
        filter_line = '%s%s,issue_number:%s' % (volume_filter, year_filter, issue_number)
```
There's 3 args, but there were only 2 format strings.